docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -51,16 +66,19 @@
 * I2C EEPROM
 * Bluetooth module via UART
 * IMU (e.g. LSM6DS33)
+* I2C port extender
+  * if port extenders are used for inputs, the interrupt pin has to be connected to an interrupt capable pin of the MCU
```

Comment:
* if port expanders are used for inputs, the interrupt output pin(s) of the port expander(s) should be connected to an interrupt capable input pin of the MCU